### PR TITLE
Hooking up the topics for Kafka correctly + starting to hook up auth

### PR DESCRIPTION
**Pull request by einari**

### CODE REVIEW

**Comment by Velcrow81** [comment]
Build is failing: No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.
Check the details on how to adjust your build configuration on: https://docs.travis-ci.com/user/common-build-problems/#Build-times-out-because-no-output-was-received

Not sure why

File: Source/Navigation/Web/src/index.html [hunk 1]
```diff
@@ -1,20 +1,50 @@
...
             ${ require('./assets/images/ifrc.svg')}
         </figure>
         ${require('./Components/navigation.html') }
-    </header>   
-</body>
-</html>
 
+        <div class="login-status">
+
+            <!-- ko if: nationalSocieties().length > 1 -->
```
Comment (Velcrow81) [inline Source/Navigation/Web/src/index.html]: Knockout?

Comment (einari) [reply Source/Navigation/Web/src/index.html]: Yup.. 
The thing is that this particular page should be very small - ideally I'd not do any framework, but Knockout had the least impact of them all - sizewize.  In the interest of time I decided to do this.

File: Source/Admin/deploy/dev-deployment.yaml [hunk 2]
```diff
@@ -44,6 +44,12 @@ spec:
         image: karolikl/cbs-admin-backend:$BUILD_ID
         ports:
         - containerPort: 80
+        env:
+        - name: "KAFKA_BOUNDED_CONTEXT_SEND_TOPICS"
+          value: "volunteerreporting;alert;usermanagement"
+        - name: "KAFKA_BOUNDED_CONTEXT_TOPIC"
+          value: "admin"
+
```
Comment (karolikl) [inline Source/Admin/deploy/dev-deployment.yaml]: The [boundedContext]/deploy/*.yaml files should be removed as all deploy files can now be found in the /Deploy/templates folder. Sorry for not having cleaned this up! 
Can you add these environment variables to https://github.com/IFRCGo/cbs/blob/master/Deploy/templates/admin.yaml instead?

File: Source/UserManagement/deploy/dev-deployment.yaml [hunk 3]
```diff
@@ -44,6 +44,12 @@ spec:
         image: karolikl/cbs-usermgmt-backend:$BUILD_ID
         ports:
         - containerPort: 80
+        env:
```
Comment (karolikl) [inline Source/UserManagement/deploy/dev-deployment.yaml]: The [boundedContext]/deploy/*.yaml files should be removed as all deploy files can now be found in the /Deploy/templates folder. Sorry for not having cleaned this up! 
Can you add these environment variables to https://github.com/IFRCGo/cbs/blob/master/Deploy/templates/usermanagement.yaml instead?

File: Source/VolunteerReporting/deploy/dev-deployment.yaml [hunk 4]
```diff
@@ -44,6 +44,12 @@ spec:
         image: karolikl/cbs-vr-backend:$BUILD_ID
         ports:
         - containerPort: 80
+        env:
```
Comment (karolikl) [inline Source/VolunteerReporting/deploy/dev-deployment.yaml]: The [boundedContext]/deploy/*.yaml files should be removed as all deploy files can now be found in the /Deploy/templates folder. Sorry for not having cleaned this up!
Can you add these environment variables to https://github.com/IFRCGo/cbs/blob/master/Deploy/templates/volunteerreporting.yaml instead?

**Comment by einari** [comment]
Added the topics configuration to the correct files now. Hoping Travis will not be stuck..

**Comment by Velcrow81** [comment]
Hope.....the best build automation 😊

**Comment by einari** [comment]
I gave it hope and in return it passed the build... I'm becoming a true believer (in hope)

**Pull request merged**